tsconfig-prod.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": "./buildtools/tsconfig-base.json",
```

Comment:
Any reason for extending and copying the content from `tsconfig.json`?
What I did on my side is to extend `./tsconfig.json` , and just override the `reference` value.